procedures/release-duty/index.rst

```diff
@@ -128,11 +128,8 @@ for how to get set up and started with connecting to the VPN.
 How to get Ship-it access
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to be added to the `vpn_cloudops_shipit` LDAP group, as well as be added to the allowedlist in the shipit
-config. Both of these can be done by filing a ticket requesting access.
-
-File a Release Engineering bug under the `Applications: Shipit (backend)
-<https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&cloned_bug_id=1626312&product=Release%20Engineering&component=Applications%3A%20Shipit%20%28backend%29>`__
+You need to be added to the `vpn_cloudops_shipit` LDAP group. File an `Infrastructure & Operations` bug under the `Infrastructure: LDAP
+<https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&cloned_bug_id=1773239&product=Release%20Engineering&component=Applications%3A%20Shipit%20%28backend%29>`__
```

Comment:
the product/component in the link are still off :)

Reply:
Oops, there we go!